### PR TITLE
Implement 1.19 support

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: server/player/player.go [hunk 1]
```diff
@@ -246,6 +246,12 @@ func (p *Player) SendJukeboxPopup(a ...any) {
 	p.session().SendJukeboxPopup(format(a))
 }
 
+// SendToast sends a toast to the player. This toast is shown at the top of the screen, similar to achievements or pack
+// loading.
+func (p *Player) SendToast(title, message string) {
```
Comment (Sandertv) [inline server/player/player.go:251]: Do you reckon we could come up with a nicer name than "toast"? It's such a shit name lmao.

Comment (provsalt) [reply server/player/player.go:251]: > Do you reckon we could come up with a nicer name than "toast"? It's such a shit name lmao.

Perhaps alert?

File: server/session/handler_player_action.go [hunk 2]
```diff
@@ -51,6 +51,8 @@ func handlePlayerAction(action int32, face int32, pos protocol.BlockPos, entityR
 			return nil
 		}
 		s.c.ContinueBreaking(cube.Face(face))
+	case protocol.PlayerActionStartItemUseOn, protocol.PlayerActionStopItemUseOn:
+		// TODO: Properly utilize these actions.
```
Comment (T14Raptor) [inline server/session/handler_player_action.go:55]: Do we have to?

File: server/session/session.go [hunk 3]
```diff
@@ -439,6 +439,7 @@ func (s *Session) registerHandlers() {
 		packet.IDPlayerAction:          &PlayerActionHandler{},
 		packet.IDPlayerAuthInput:       &PlayerAuthInputHandler{},
 		packet.IDPlayerSkin:            &PlayerSkinHandler{},
+		packet.IDRequestAbility:        nil, // TODO: Properly support this.
```
Comment (Sandertv) [inline server/session/session.go:442]: Think you can remove the TODO for this, we don't really need to handle this at any point in the near future.

**Review by Sandertv — COMMENTED** [review]
Should be good to go once 1.19 is released!

File: server/session/handler_player_action.go (hunk 2 above)
Comment (JustTalDevelops) [reply server/session/handler_player_action.go:55]: It would be ideal in my opinion. @Sandertv thoughts?

File: server/session/session.go (hunk 3 above)
Comment (JustTalDevelops) [reply server/session/session.go]: Removed.

**Pull request merged**